packages/indexer-common/src/indexer-management/client.ts

```diff
@@ -222,6 +276,15 @@ const SCHEMA_SDL = gql`
 
     storeDisputes(disputes: [POIDisputeInput!]!): [POIDispute!]
     deleteDisputes(allocationIDs: [String!]!): Int!
+
+    createAllocation(deploymentID: String!, amount: String!): CreateAllocationResult!
```

Comment:
I'd like to see the ability to specify an override for the `index-node` assignment when creating or reallocating. The reason for this is that some subgraphs have unusual performance profiles (which impacts indexing performance of colocated deployments), and it's desirable to allocate these on a specific `index-node`.

Reply:
Good idea, I'll add an optional string parameter for specifying an index-node.